### ISSUE TITLE
Stop using Nginx

### ISSUE THREAD

**Issue by nyfair** [stop-war-in-ukraine/stop-russia-it#51]
It's a shame to host website by Nginx, a generic proxy server from Russia

**Comment by vladimirlapskiy** [comment]
So nginx already owned by the company F5 (F5 From United States). 
Also from wikipedia:
On 18 January 2022, it was announced that Igor Sysoev is leaving Nginx and F5.
(https://en.wikipedia.org/wiki/Nginx#History)

**Comment by digiom-ceo** [comment]
Bought for dollars and subsequently the dollars were taken away. Lol. America has no faith. This is a nginx raider takeover.

**Comment by jsisaev1** [comment]
обиженки из ОПГ ЛГБТ, чего всполошились? 
ваш софт (украинский) полное говно. что гитлаб шлак, что еще масса примеров по типу macpaw.
так что денацифицируйтесь, нормально учитесь и приезжайте целовать Русский сапог.
вы ведь этого добиваетесь?
вон мишустин обнулил НДС и отменил нологи ИТ-компаниям, а также освободил от армии всех разработчиков и одобрил еботеку под 5%.
так что приезжайте к нам работать, может что толковое из вас тут выйдет

**Comment by nyfair** [comment]
> Bought for dollars and subsequently the dollars were taken away. Lol. America has no faith. This is a nginx raider takeover.

Moreover, America should return Alaska since it was bought from evil land

**Comment by jsisaev1** [comment]
> > Bought for dollars and subsequently the dollars were taken away. Lol. America has no faith. This is a nginx raider takeover.
> 
> Moreover, America should return Alaska since it was bought from evil land

yes. also texas is a part of Mexico and western part of USA is French.

**Comment by greatbody** [comment]
> So nginx already owned by the company F5 (F5 From United States). Also from wikipedia: "On 18 January 2022, it was announced that Igor Sysoev is leaving Nginx and F5." (https://en.wikipedia.org/wiki/Nginx#History)

How could an US company pay money to Putin and support the Russia regime.
They are traitors of the great US.
